src/libs/actions/IOU.ts

```diff
@@ -3094,10 +3095,10 @@ function categorizeTrackedExpense(
     taxCode = '',
     taxAmount = 0,
     billable?: boolean,
-    receipt?: Receipt,
+    receipt: OnyxEntry<Receipt> = null,
```

Comment:
```suggestion
    receipt: Receipt | null = null,
```

`Receipt` is not a value stored in Onyx